bot/admin/web/src/app/bot-admin-app.component.ts

```diff
@@ -89,7 +89,7 @@ export class BotAdminAppComponent implements AuthListener, OnInit, OnDestroy {
         title: 'Settings',
         icon: 'settings-outline',
         link: '/configuration',
-        hidden: !this.state.hasRole(UserRole.admin)
+        hidden: !this.state.hasRole(UserRole.botUser) && !this.state.hasRole(UserRole.faqBotUser)
```

Comment:
We should open this menu entry to any user without restrictions. All users need to be able to switch namespace.

I would simply set `hidden: false` with a comment indicating that all users can switch namespace.

Reply:
Done